cmd/run.go

```diff
@@ -125,19 +125,21 @@ a commandline interface for interacting with it.`,
 		// defaults in there, override with Runner-provided ones, then merge the CLI opts in
 		// on top to give them priority.
 		fmt.Fprintf(stdout, "%s options\r", initBar.String())
-		cliConf, err := getConfig(cmd.Flags())
+
+		defaultConf := Config{Options: r.GetOptions()}
```

Comment:
Ah, unfortunately this would break some stuff :unamused:... We have to use the `cliConf` as the base from which to build the config (and also `Apply()` it last) because it both contains the [default values](https://github.com/loadimpact/k6/blob/master/cmd/options.go#L47-L66) for different options AND is the highest user-supplied config set in the configuration hierarchy... 
For example if you try running the code as it is without specifying VUs anywhere, it won't run because the number of VUs would be `0`.
I'm sorry you have to deal with this, we'll hopefully fix the configuration mess soon. I'm still collecting pain points, issues and unexpected behaviours, but I'll soon create a github issue that comprehensively describes the current problems with option and configuration handling. For the moment, to avoid breakage, please don't touch the sacred :hankey:  that is `cliConf.Apply(fileConf).Apply(Config{Options: r.GetOptions()}).Apply(envConf).Apply(cliConf)` :unamused: